backend/models/user.py

```diff
@@ -3,35 +3,48 @@
 from typing import Optional, List
 from pydantic import BaseModel, validator, constr
 
+
+class UserRole(str, Enum):
+    ANGANVADI = 'ANGANVADI'
+    NGO_MEMBER = 'NGO_MEMBER'
+    RELATIVE = 'RELATIVE'
+
+
 class UserType(str, Enum):
     PREGNANT = 'PREGNANT'
     LACTATING = 'LACTATING'
     CAREGIVER = 'CAREGIVER'
 
+
 class GenderType(str, Enum):
     MALE = 'MALE'
     FEMALE = 'FEMALE'
 
+
 class Child(BaseModel):
     name: str
     gender: GenderType
     dob: datetime.date
+
     @validator("dob", pre=True)
     def parse_dob(cls, value):
         return datetime.datetime.strptime(
             value,
             "%d/%m/%Y"
         ).date()
 
+
 class CreateUserSchema(BaseModel):
     name: str
     phone_number: constr(
         strip_whitespace=True,
         regex=r"^(\+)[1-9][0-9\-\(\)\.]{9,15}$",
     )
     user_type: UserType
+    user_role: UserRole
     child: List[Child]
     relation_with_child: Optional[str]
+    is_created_for_someone_else: Optional[bool] = False
```

Comment:
relation_with_child field is not required anymore if you are using user_role for that. You can remove that.
On the separate note, relation_with_child is more readable field so I would suggest you to rename "user_role" to "relation_with_child" . What do you think ?

Reply:
Yes I agree with you and  I will rename that